packages/app/pages/Organization/flexboxstyle.scss

```diff
@@ -0,0 +1,33 @@
+/*- Flex Box Styling -*/
+
+img_style {
+  width: 60px;
+  height: 60px
+}
+
+flex_container {
+  display: flex;
+  background_color: white;
```

Comment:
Another syntax error

Reply:
This has been resolved.  This entire file is not used anywhere so it has been removed.